gui/src/styles.scss

```diff
@@ -110,3 +118,24 @@ body {
     padding-right: 0 !important;
   }
 }
+//font for bold
+.font-thick {
+  font-weight: 500 !important;
+}
+
+.material-icons.orange {
+  color: #FB8C00;
+  vertical-align: bottom;
+}
+.material-icons.red {
+  color: #ff0000;
+  vertical-align: bottom;
+}
+.material-icons.green {
+  color: #38ff80;
```

Comment:
This green is too light and it's hard to recognize it. Choose some darker green, e.g.: `#28a745`